### ISSUE TITLE
Storybook fails to build from InterpolateHtmlPlugin error [alpha.15]

### ISSUE THREAD

**Issue by IanVS** [storybooks/storybook#3953]
If you are reporting a bug or requesting support, start here:
### Bug or support request summary

I tried to upgrade from `alpha.12` to `alpha.15`, and now when I perform a `build-storybook`, I'm getting the error:

```
ERR! Failed to build the storybook
ERR!   TypeError: callback is not a function
ERR!
ERR!   - InterpolateHtmlPlugin.js:37 compilation.plugin
ERR!     [web]/[react-dev-utils]/InterpolateHtmlPlugin.js:37:11
.
.
.
```

### Steps to reproduce
Update to `alpha.15`, perform a `build-storybook`.  That's all I did, but maybe there are other contributing factors?

### Please specify which version of Storybook and optionally any affected addons that you're running

	"@storybook/addon-actions": "4.0.0-alpha.15",
    "@storybook/addon-info": "4.0.0-alpha.15",
    "@storybook/addon-knobs": "4.0.0-alpha.15",
    "@storybook/addon-options": "4.0.0-alpha.15",
    "@storybook/react": "4.0.0-alpha.15",

### Affected platforms 

```
❯ yarn list --pattern react-dev-utils
yarn list v1.9.2
└─ react-dev-utils@6.0.0-next.b2fd8db8
```

It seems like maybe this could be related to https://github.com/storybooks/storybook/pull/3852.

**Comment by igor-dv** [comment]
Until the fix is released, I can suggest locking manually a right version of the `react-dev-utils` (3e165448) in your lock file.

**Comment by strothj** [comment]
Pinning the version to `3e165448` allows my project to build but then the command line hangs up after completion.

`package.json`:
```
{
  "resolutions": {
    "react-dev-utils": "6.0.0-next.3e165448"
  }
}
```

```
PS C:\project-path> yarn storybook:build
yarn run v1.9.4
$ build-storybook -c .storybook -s public -o dist/storybook
info @storybook/react v4.0.0-alpha.15
info
info => Loading custom addons config.
WARN => Custom configuration file C:\project-path\.storybook\webpack.config.js is detected
WARN    but impossible to import loader for .js
info => Loading custom webpack config (full-control mode).
info => Copying static files from: public
info Building storybook ...
info Building storybook completed.
Terminate batch job (Y/N)? y
```

4.0.0-alpha.14 continues to work fine for me.

**Comment by ndelangen** [comment]
Do you have a method so I can reproduce?

Either way that plugin is being replaced/removed:
https://github.com/storybooks/storybook/pull/3919

**Comment by ndelangen** [comment]
This change might also fix it: https://github.com/storybooks/storybook/pull/3919

**Comment by pksunkara** [comment]
@ndelangen No, only #3959 would fix it. `WatchMissingNodeModulesPlugin` also produces this issue.

**Comment by shilman** [comment]
Released a fix in [4.0.0-alpha.16](https://github.com/storybooks/storybook/releases/tag/v4.0.0-alpha.16)
